report.sh

```diff
@@ -32,12 +32,12 @@ echo "Choose an option below!
 echo " "
 echo -n "Chosen option: "
 read opcao
-case $opcao in
+case "${opcao}" in
 	1)
 		function processador () {
-			CPU_INFO=`cat /proc/cpuinfo | grep -i "^model name" | cut -d ":" -f2 | sed -n '1p'`
-			echo "CPU model: $CPU_INFO"
-			sleep $TIME
+            CPU_INFO=$(cat /proc/cpuinfo | grep -i "^model name" | cut -d ":" -f2 | sed -n '1p')
```

Comment:
You are WAY too young to be allowed to style use backquote for command substitution ;)

Reply:
Side note: look at PIPE_STATUS to learn how to check that a piped commands were successful (or not)

Reply:
okay sure! I'll take a look at PIPE_STATUS
The backquote style I have learned from here :p https://www.gnu.org/software/bash/manual/html_node/Command-Substitution.html